src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figure;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(getDeskription());
+    }
+
+    public String getDeskription() {
```

Comment:
1)
```suggestion
    public String getDescription() {
```
2) In case you want to create this getDescription, you better make an abstract method for that, don't you think?